ios/Classes/TiMapView.m

```diff
@@ -1134,7 +1138,7 @@ - (MKAnnotationView *)mapView:(MKMapView *)mapView viewForAnnotationProxy:(TiMap
   if (customView == nil && !marker) {
     id imagePath = [ann valueForUndefinedKey:@"image"];
     image = [TiUtils image:imagePath proxy:ann];
-    identifier = (image != nil) ? @"timap-image" : @"timap-pin";
+    identifier = (image != nil) ? @"timap-image" : @"carIcon"; //carIcon
```

Comment:
@anil-shukla-axway , Why did you changed 'timap-pin' to `carIcon` ?

Reply:
Updated the name, Please verfiy.